### PR TITLE
Fixed building error

### PR DESCRIPTION
**Pull request by YuriAzathoth**
Signed-off-by: YuriAzathoth <iamkiller@mail.com>

### CODE REVIEW

File: include/OgreOggSoundPrereqs.h [hunk 1]
```diff
@@ -49,7 +48,7 @@
  * 1 - Enable EFX suport
  */
 #	ifndef HAVE_EFX
-#		define HAVE_EFX 1
+#		define HAVE_EFX 0
```
Comment (paroj) [inline include/OgreOggSoundPrereqs.h]: is this change on purpose, or was it accidentally included?

Comment (YuriAzathoth) [reply include/OgreOggSoundPrereqs.h]: Sorry,
It does not a change on purpose, it is my fault on comitting.
Thanks, I did not noticed that.

Comment (YuriAzathoth) [reply include/OgreOggSoundPrereqs.h]: I changed one dependency from OpenAL to OpenALSoft, and I do not know, why OpenALSoft does not suport EFX.
But CMakeList.txt must work.

**Comment by paroj** [comment]
applied the cleaner patch referenced by @Asmodeus666
